Update issue-solution.md (#777)
mattermost/mattermost-handbook: operations/operations/issue-solution.md

# Issue/Solution Process

> Issue/Solution Proposal is a tool for increasing clarity, speed and output in teams. 

Reviewer: ```suggestion
Issue/Solution Proposal is a tool for increasing clarity, speed, and output in teams.
```

> We are now on Version 3 of our process and the documentation, video training and ISP template are hosted internally here: https://docs.google.com/document/d/17oHNwYMg_iil3HpGgg1QBIqCX4Q9KoYAyqMvjfwTiGE/edit

Reviewer: ```suggestion
We're now on version three of our process. The documentation, video training, and ISP template are hosted internally [here](https://docs.google.com/document/d/17oHNwYMg_iil3HpGgg1QBIqCX4Q9KoYAyqMvjfwTiGE/edit).
```

> Over time, this page in the handbook will be replaced with a [Focalboard](https://focalboard.com) page where a summary of our process and template will be hosted. 

Reviewer: ```suggestion
Over time, this page will be replaced with a [Focalboard](https://focalboard.com) page where a summary of our process and template will be hosted.
```
